openassessment/assessment/models/ai.py

```diff
@@ -25,18 +25,12 @@
 # The classifier data will generally be larger than memcached's default max size
 CLASSIFIERS_CACHE_IN_MEM = getattr(
     settings, 'ORA2_CLASSIFIERS_CACHE_IN_MEM',
-    get_cache(
-        'django.core.cache.backends.locmem.LocMemCache',
-        LOCATION='openassessment.ai.classifiers_dict'
-    )
+    caches['cache_in_mem']
 )
 
 CLASSIFIERS_CACHE_IN_FILE = getattr(
     settings, 'ORA2_CLASSIFIERS_CACHE_IN_FILE',
-    get_cache(
-        'django.core.cache.backends.filebased.FileBasedCache',
-        LOCATION='/tmp/ora2_classifier_cache'
-    )
+    caches['cache_in_file']
```

Comment:
The above changes are working in ora2 but not working in edx-platform because `ora2` settings will not be available there due to which we will get `InvalidCacheBackendError: Could not find config for 'cache_in_mem' in settings.CACHES`. So we have two options:
1. leave the old code as is
2. use [_create_cache](https://github.com/django/django/blob/1.8.4/django/core/cache/__init__.py#L65) function

The issue with 2nd option is that thread safety is not guaranteed but [get_cache in 1.4](https://github.com/django/django/blob/1.4.22/django/core/cache/__init__.py#L141) is already thread unsafe and working fine

@nedbat @symbolist @doctoryes @alawibaba @symbolist @macdiesel Suggestions are welcome. Thanks!